src/js/UI-remove-task.js

```diff
@@ -0,0 +1,10 @@
+export default class RemoveActivities {
+  static removeAct(target) {
+    // target.parentNode.parentNode.RemoveActivities(target.parentNode)
+    target.parentNode.parentNode.removeChild(target.parentNode);
```

Comment:
- [ ] You have effectively implemented the functionalities of this app. However, I would suggest you remove all commented codes to maintain a clean codebase thus complying with the YANGI principle